tests/TestApplication/src/Entity/Order.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TestApp\Entity;
+
+use CommerceWeavers\SyliusTpayPlugin\Model\OrderLastNewPaymentAwareInterface;
+use CommerceWeavers\SyliusTpayPlugin\Model\OrderLastNewPaymentAwareTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Sylius\Component\Core\Model\Order as BaseOrder;
+
+#[ORM\Entity]
+#[ORM\Table(name: 'sylius_order')]
+class Order extends BaseOrder implements OrderLastNewPaymentAwareInterface
+{
+    use OrderLastNewPaymentAwareTrait;
```

Comment:
later, would be good to ensure, that in the place we are using functions from this trait we are not assuming that it will always be tpay